downstream-templates/python/README.md

```diff
@@ -67,4 +67,4 @@ except ApiException as e:
     print("Got an exception: {}".format(e))
 ```
 
-For a working example, see [the example folder](./example) in this repo.
+For a working example, see [the example folder](https://github.com/arduino/iot-client-py/tree/master/example) in this repo.
```

Comment:
please point directly to https://github.com/arduino/iot-client-py/blob/master/example/main.py